docs/guides/setup/configuring-data-docs/how-to-host-and-share-data-docs-on-gcs.md

```diff
@@ -1,5 +1,171 @@
+  app = Flask(__name__)
+  # Configure this environment variable via app.yaml
+  CLOUD_STORAGE_BUCKET = os.environ['CLOUD_STORAGE_BUCKET']
+  @app.route('/', defaults={'path': 'index.html'})
+  @app.route('/<path:path>')
+  def index(path):
+      gcs = storage.Client()
+      bucket = gcs.get_bucket(CLOUD_STORAGE_BUCKET)
+      try:
+          blob = bucket.get_blob(path)
+          content = blob.download_as_string()
+          if blob.content_encoding:
+              resource = content.decode(blob.content_encoding)
+          else:
+              resource = content
+      except Exception as e:
+          logging.exception("couldn't get blob")
+          resource = "<p></p>"
+      return resource
+  @app.errorhandler(500)
+  def server_error(e):
+      logging.exception('An error occurred during a request.')
+      return """
+      An internal error occurred: <pre>{}</pre>
+      See logs for full stacktrace.
+      """.format(e), 500
+  ```
+
+3. **If you haven't done so already, authenticate the gcloud CLI and set the project.**
+
+  ```bash
+  # Insert the appropriate project name.
+
+  > gcloud auth login && gcloud config set project <<project_name>>
+  ```
+
+4. **Deploy your Google App Engine app.**
+
+  Issue the following CLI command from within the app directory created above: ``gcloud app deploy``.
+
+5. **Set up Google App Engine firewall for your app to control access.**
+
+  Visit the following page for instructions on creating firewall rules: [Creating firewall rules](https://cloud.google.com/appengine/docs/standard/python3/creating-firewalls)
+
+6. **Add a new GCS site to the data_docs_sites section of your great_expectations.yml.**
+
+  You may also replace the default ``local_site`` if you would only like to maintain a single GCS Data Docs site.
+
+  ```yaml
+  data_docs_sites:
+    local_site:
+      class_name: SiteBuilder
+      show_how_to_buttons: true
+      store_backend:
+        class_name: TupleFilesystemStoreBackend
+        base_directory: uncommitted/data_docs/local_site/
+      site_index_builder:
+        class_name: DefaultSiteIndexBuilder
+    gs_site:  # this is a user-selected name - you may select your own
+      class_name: SiteBuilder
+      store_backend:
+        class_name: TupleGCSStoreBackend
+        project: my_org_project # UPDATE the project name with your own
+        bucket: my_org_data_docs  # UPDATE the bucket name here to match the bucket you configured above
+      site_index_builder:
+        class_name: DefaultSiteIndexBuilder
+  ```
+
+7. **Build the GCS Data Docs site.**
+
+  Use the following CLI command: ``great_expectations --v3-api docs build --site-name gs_site``. If successful, the CLI will provide the object URL of the index page. Since the bucket is not public, this URL will be inaccessible. Rather, you will access the Data Docs site using the App Engine app configured above.
+
+  ```bash
+  > great_expectations --v3-api docs build --site-name gs_site
```

Comment:
@cdkini Not sure here -- do we need the "greater-than" sign?  Was this supposed to be "a terminal/shell prompt" of sorts?  Thanks!

Reply:
This was in RTD but I've updated per your comment!